group/assignment5/main.cpp

```diff
@@ -0,0 +1,94 @@
+#include <string>
+#include "test.hpp"
+#include "orgraph.hpp"
+
+std::vector<char> get_alphabet_from_dict (std::vector<std::string> dict)
+{
+    if (dict.empty())
+        return std::vector<char>();
+    std::vector<char> sym_list;
+    std::vector<int> sym_to_id (256, -1);
```

Comment:
You can add this line at the beginning of your file and avoid writing std:: all the time :) 

using namespace std;